tests/test_edf.py

```diff
@@ -527,6 +527,73 @@ def test_edf_signal_update_data_keep_physical_range_raises_error_if_new_data_exc
         signal.update_data(np.array(data), keep_physical_range=True)
 
 
+def test_edf_signal_update_data_resampling():
+    edf = Edf(
```

Comment:
do we need the `Edf` object here?

Reply:
No, we don't simplified the tests in [e90fdb2](https://github.com/the-siesta-group/edfio/pull/13/commits/e90fdb2977023a28943f0bbc0efa7f12a5cf0d62).